docs/azure/aksextensions.md

```diff
@@ -141,6 +145,17 @@ Right-click on your AKS cluster and select **Show Properties** to display the AK
 
 ![Start or Stop Cluster from properties webview](images/aksextensions/aks-startstop-cluster.png)
 
+### Run Kubectl Commands from your AKS cluster
+
+Right click on your AKS cluster and click on **Run Kubectl Commands** to easily run few known kubectl commands on your cluster. Currently we have enable following kubectl commands for the AKS cluster:
```

Comment:
"Right-click on your AKS cluster and select **Run KubectlCommands** to run Kubectl commands on your cluster. Currently, the following Kubectl commands are available:"